TicketSwap/solar
# Add CopyLink icon

# Description

This PR adds a Copy link icon

## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Go to `http://localhost:9001/?path=/story/guidelines-iconography--page`
- Check if the CopyLink icon is there

## Screenshots

![image](https://user-images.githubusercontent.com/12168698/146005892-0eab396c-7957-4bdf-b44f-1f71b8da0ade.png)
